djangocms_blog/models.py

```diff
@@ -578,6 +578,13 @@ class Meta:
     objects = GenericDateTaggedManager()
     admin_manager = AdminDateTaggedManager()
 
+    @property
```

Comment:
I do not think this should be needed? When did you encounter a connected issue?

Reply:
This turns out to be a bug in djangocms-versioning. Fixed here: https://github.com/django-cms/djangocms-versioning/pull/423